worlds/ffmq/Client.py

```diff
@@ -47,6 +47,17 @@ def get_flag(data, flag):
     bit = int(0x80 / (2 ** (flag % 8)))
     return (data[byte] & bit) > 0
 
+def validate_read_state(data1, data2):
+    validation_array = bytes([0x01, 0x46, 0x46, 0x4D, 0x51, 0x52])
```

Comment:
```suggestion
    validation_array = b"\x01FFMQR"
```
another way this could be written (not saying one should be preferred over the other)

Reply:
```python
>>> a = bytes([0x01, 0x46, 0x46, 0x4D, 0x51, 0x52])
>>> b = b"\x01FFMQR"
>>> a == b
True
```

Reply:
good to know, i'll keep that in mind for next time